src/job_cache/types.cpp

```diff
@@ -490,16 +628,16 @@ JAST FindJobRequest::to_json() const {
   return json;
 }
 
-FindJobResponse::FindJobResponse(JAST json) {
+FindJobResponse::FindJobResponse(const std::string &client_cwd, JAST json) {
   JAST found = json.get("found");
   if (found.kind != JSON_TRUE) {
     match = {};
     return;
   }
-  match = wcl::make_some<MatchingJob>(json.get("match"));
+  match = wcl::make_some<MatchingJob>(client_cwd, json.get("match"));
 }
 
-JAST FindJobResponse::to_json() const {
+JAST FindJobResponse::to_json(const std::string &client_cwd) const {
```

Comment:
Can `client_cwd` here be stored in the constructor since it's provided there?

Reply:
oh good question, let me think about that.

Reply:
yep that worked. I was trying to avoid adding info that didn't need to be there but it doesn't really matter too much because we're just sending back the thing that the user told us so they can ignore it if they want. It might be useful for debugging in the future too.